apps/judicial-system/api/src/app/modules/court/court.spec.ts

```diff
@@ -0,0 +1,200 @@
+import { uuid } from 'uuidv4'
+import { formatISO } from 'date-fns' // eslint-disable-line no-restricted-imports
+
+import { Test } from '@nestjs/testing'
+
+import { LoggingModule } from '@island.is/logging'
+import { CaseType, User } from '@island.is/judicial-system/types'
+import { AuditTrailModule } from '@island.is/judicial-system/audit-trail'
+import {
+  AuthenticateApi,
+  CreateCaseApi,
+  CreateCustodyCaseApi,
+} from '@island.is/judicial-system/court-client'
+
+import { BackendAPI } from '../../../services'
+import { environment } from '../../../environments'
+import { CourtResolver } from './court.resolver'
+import { CourtService } from './court.service'
+
+function expectReceivalDate(receivalDate: string) {
+  const date = new Date()
+  const today = formatISO(date, { representation: 'date' })
+  date.setDate(date.getDate() - 1)
+  const yesterday = formatISO(new Date(), { representation: 'date' })
```

Comment:
Isn't the `new Date()` here supposed to be `date`? Otherwise, I think this is a confusing variable name in that the value is actually today 🤔

Reply:
You are right. Will fix.

Reply:
Well spotted.